automatic/selenium.powershell/README.md

```diff
@@ -0,0 +1,20 @@
+# Selenium PowerShell Module
+
+The Selenium PowerShell module allows you to automate browser interaction using the [Selenium API](https://selenium.dev/). You can navigate to pages, find elements, click buttons, enter text and even take screenshots. 
+
+- Wraps the C# WebDriver for Selenium
+- Easily execute web-based tests
+- Works well with Pester
```

Comment:
These sounds like they should be in a `Features` section, see the Contribution guidelines: https://github.com/chocolatey-community/chocolatey-packages/blob/master/CONTRIBUTING.md#description